### PR TITLE
Fixes SiCKRAGETV/sickrage-issues#2584

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Had to redo my branch as I was accidently working from the wrong dev branch.

### CODE REVIEW

File: gui/slick/interfaces/default/home.mako [hunk 1]
```diff
@@ -770,11 +781,9 @@ $(document).ready(function(){
     % if layout != 'simple':
         <td align="center">
         % if curShow.network:
-            <span title="${curShow.network}" class="hidden-print"><img id="network" width="54" height="27" src="${sbRoot}/showNetworkLogo/?show=${curShow.indexerid}" alt="${curShow.network}" title="${curShow.network}" /></span>
-            <span class="visible-print-inline">${curShow.network}</span>
+            <span title="${curShow.network}"><img id="network" width="54" height="27" src="${sbRoot}/showNetworkLogo/?show=${curShow.indexerid}" alt="${curShow.network}" title="${curShow.network}" /></span>
```
Comment (MGaetan89) [inline gui/slick/interfaces/default/home.mako]: This should not have changed

File: gui/slick/interfaces/default/home.mako [hunk 2]
```diff
@@ -770,11 +781,9 @@ $(document).ready(function(){
...
         % else:
-            <span title="No Network" class="hidden-print"><img id="network" width="54" height="27" src="${sbRoot}/images/network/nonetwork.png" alt="No Network" title="No Network" /></span>
-            <span class="visible-print-inline">No Network</span>
+            <span title="No Network"><img id="network" width="54" height="27" src="${sbRoot}/images/network/nonetwork.png" alt="No Network" title="No Network" /></span>
```
Comment (MGaetan89) [inline gui/slick/interfaces/default/home.mako]: This should not have changed

File: gui/slick/interfaces/default/home.mako [hunk 3]
```diff
@@ -790,36 +799,12 @@ $(document).ready(function(){
...
-                        classtoadd = "progress-80"
-                    }
-                    if (classvalue==100) {
-                        classtoadd = "progress-100"
-                    }
-                    $("#progressbar${curShow.indexerid} > .ui-progressbar-value").addClass(classtoadd);
-                });
-            </script>
+            <span style="display: none;">${download_stat}</span>
+            <div class="progressbar" style="position:relative;" data-show-id="${curShow.indexerid}" data-progress-percentage="${progressbar_percent}" data-progress-text="${download_stat}" data-progress-tip="${download_stat_tip}"></div>
```
Comment (MGaetan89) [inline gui/slick/interfaces/default/home.mako]: This need a `hidden-print` class as well.

File: gui/slick/interfaces/default/home.mako (hunk 1 above)
Comment (OmgImAlexis) [inline gui/slick/interfaces/default/home.mako]: I know that, check the last commit.

**Comment by MGaetan89** [comment]
Looking good now :+1: 
Can't the JS starting line 532 be removed too? I don't think it works any more, as you removed the id.

**Comment by OmgImAlexis** [comment]
@MGaetan89 I wasn't even sure what that part was for, the mako file is serious too big with too many if statements.

If that's just for progress bars like the other one then it can definitely be removed. @miigotu can you confirm this?

**Comment by MGaetan89** [comment]
I spent quite some time on that file lately, so I know what it does :p
This part is for the progress in the "Poster" layout. This only difference with the other progress, it's that the stats are not displayed on top of it. So you might need to add an extra argument for that.

**Comment by OmgImAlexis** [comment]
For now I'll leave it but I think I know what I'm going to do. We should be able to move some of those if statements around and remove that js for the `progressbar` instead of having duplicate content like we have at the moment.

**Comment by MGaetan89** [comment]
I agree.
But I think you should restore the id in "Poster" layout if you are not going to remove the JS. Otherwise it won't work I guess (I can't actually try for the moment, so I'm not 100% sure).

**Comment by OmgImAlexis** [comment]
@MGaetan89 there we go, the other js is removed and now uses a ternary statement to check if the `data-progress-text` is set, if it isn't then it doesn't append the text.e

**Comment by fernandog** [comment]
@OmgImAlexis sorting using progress bar it's not working

also 69+1 shouldn't be GREEN

image

**Comment by OmgImAlexis** [comment]
@fernandog Working on that now.

**Comment by OmgImAlexis** [comment]
This should fix it. https://github.com/SiCKRAGETV/SickRage/pull/2365

The green bar isn't anything I changed. The progress is using `${progressbar_percent}` which is still using this function from above.

````
cur_airs_next = ''
    cur_airs_prev = ''
    cur_snatched = 0
    cur_downloaded = 0
    cur_total = 0
    download_stat_tip = ''

    if curShow.indexerid in show_stat:
        cur_airs_next = show_stat[curShow.indexerid]['ep_airs_next']
        cur_airs_prev = show_stat[curShow.indexerid]['ep_airs_prev']

        cur_snatched = show_stat[curShow.indexerid]['ep_snatched']
        if not cur_snatched:
            cur_snatched = 0

        cur_downloaded = show_stat[curShow.indexerid]['ep_downloaded']
        if not cur_downloaded:
            cur_downloaded = 0

        cur_total = show_stat[curShow.indexerid]['ep_total']
        if not cur_total:
            cur_total = 0

    if cur_total != 0:
        download_stat = str(cur_downloaded)
        download_stat_tip = "Downloaded: " + str(cur_downloaded)
        if cur_snatched > 0:
            download_stat = download_stat + "+" + str(cur_snatched)
            download_stat_tip = download_stat_tip + "&#013;" + "Snatched: " + str(cur_snatched)

        download_stat = download_stat + " / " + str(cur_total)
        download_stat_tip = download_stat_tip + "&#013;" + "Total: " + str(cur_total)
    else:
        download_stat = '?'
        download_stat_tip = "no data"

    nom = cur_downloaded
    den = cur_total
    if den == 0:
        den = 1

    progressbar_percent = nom * 100 / den
````

**Pull request merged**